src/pages/AboutPage.js

```diff
@@ -38,8 +42,8 @@ const AboutPage = () => (
 
     <hr />
     
-    <Row>
-      <Col>
+    <Row className="align-center">
+      <Col className>
```

Comment:
was this classname forgotten?

Reply:
ah my bad. was going through and removing classes. fixing now (noticed it on another line too)